src/kirby/components/badge/badge.component.scss

```diff
@@ -11,6 +11,7 @@
     --padding-end: 5px;
     --padding-bottom: 4px;
     --padding-start: 5px;
+    box-sizing: border-box; //Fixes https://github.com/kirbydesign/designsystem/issues/537
```

Comment:
comment: https://github.com/kirbydesign/designsystem/issues/537

If the bug not closed with ref to PR in kirby.